core/store/src/trie/mod.rs

```diff
@@ -202,12 +201,13 @@ pub struct Trie {
     /// (which can be toggled on the fly), trie nodes that have been looked up
     /// once will be guaranteed to be cached, and further reads to these nodes
     /// will encounter less gas cost.
-    accounting_cache: RefCell<TrieAccountingCache>,
+    accounting_cache: TrieAccountingCache,
     /// If present, we're capturing all trie nodes that have been accessed
     /// during the lifetime of this Trie struct. This is used to produce a
     /// state proof so that the same access pattern can be replayed using only
     /// the captured result.
-    recorder: Option<RefCell<TrieRecorder>>,
+    // FIXME: make `TrieRecorder` internally MT-safe, instead of locking the entire structure.
```

Comment:
nit: I'm not sure that MT is widely understood abbreviation, can you expand it in the comments?

Reply:
`MT-Safe` is a a documented property [in the POSIX standard](https://www.gnu.org/software/libc/manual/html_node/POSIX-Safety-Concepts.html), has its own manpage at `attributes(7)` and is difficult to expand into just a few words, unfortunately.

Though at the same time, I guess, the `MT-Safe` is not exactly the qualifier I was looking for -- granular locking is perhaps the more appropriate one.

Reply:
Maybe just "multi-thread" in parens?